docs/cli/axelard_tx_vesting_create-vesting-account.md

```diff
@@ -6,7 +6,7 @@ Create a new vesting account funded with an allocation of tokens.
 
 Create a new vesting account funded with an allocation of tokens. The
 account can either be a delayed or continuous vesting account, which is determined
-by the '--delayed' flag. All vesting accouts created will have their start time
+by the '--delayed' flag. All vesting accounts created will have their start time
```

Comment:
this one is a type in the cosmos-sdk and gets auto-generated. For the checks to pass you need to revert it